server/db/model/code_snippets/model: server.js

```diff
@@ -0,0 +1,33 @@
+const { Model, DataTypes } = require('sequelize');
+const db = require('../../db');
+
+class Server extends Model {}
```

Comment:
Hi Maciej! I'm a little confused about this model, "Server", as well as where it's located in the file structure. Why is it called "Server"? For the file structure, am I reading it correctly that there seems to be a model directory within db, and then a code_snippets directory within that model directory that houses this model file?